userguide/jails.rst

```diff
@@ -128,21 +107,21 @@ seen in :numref:`Figure %s <jail_wizard_fig>`.
 
 
 The wizard demonstrates the simplest process to create and configure
-networking for a new jail. Enter a :guilabel:`Jail Name`. It can only
-contain alphanumeric characters (:kbd:`abc`..., :kbd:`123`...), dashes
-(:kbd:`-`), and underscores (:kbd:`_`). Choose the version of FreeBSD
-to install for this jail. Previously downloaded versions display
-:literal:`(fetched)` next to their entry in the list. These settings
-are the minimum required settings for a new jail, but configuring
-recommended to also configure networking for the jail.
+networking for a new jail. Enter a :guilabel:`Jail Name`. Jail names can
+only contain alphanumeric characters (:literal:`Aa-Zz`, :literal:`123`),
+dashes (:literal:`-`), and underscores (:literal:`_`). Choose the version
+of FreeBSD to install for this jail. Previously downloaded versions
+display :literal:`(fetched)` next to their entry in the list. These are
+the minimum required settings for a new jail but it is also recommended
```

Comment:
This is repeated elsewhere in confused form.

Please don't say "it is recommended", that is nonspecific in several ways.  Better to split that sentence:
These are the minimum settings required for a new jail.
Configuring networking for new jails is also recommended.